chart/values.yaml

```diff
@@ -44,6 +44,9 @@ csv:
     features.operators.openshift.io/csi: "false"
 image: quay.io/maistra-dev/sail-operator:0.2-latest
 imagePullPolicy: Always
+proxy:
+  image: gcr.io/kubebuilder/kube-rbac-proxy:v0.16.0
+  imagePullPolicy: Always
```

Comment:
Do we want the default to be `Always`? Previously, we didn't set `imagePullPolicy`, which means that the policy was effectively `IfNotPresent`, given that the image tag was not `latest`.

Reply:
@luksa:  Good point that the behavior will be changed based on the existing absence of a value.  I followed the existing pattern but do understand that its using an explicit tag value rather than some form of _latest_.  Happy to change to IfNotPresent if you'd prefer.

Reply:
Actually, I feel like it should be left unset. But since the same should also be true for the `imagePullPolicy` used in the main container, I'm okay with merging this as is. We should then set both to empty by default.